src/main/java/org/acra/ACRA.java

```diff
@@ -269,25 +269,20 @@ public static boolean isInitialised() {
      * @return true if the current process is the process running the SenderService.
      *          NB this assumes that your SenderService is configured to used the default ':acra' process.
      */
-    public static boolean isACRASenderServiceProcess(@NonNull Application app) {
-        final String processName = getCurrentProcessName(app);
+    public static boolean isACRASenderServiceProcess() {
+        final String processName = getCurrentProcessName();
         if (ACRA.DEV_LOGGING) log.d(LOG_TAG, "ACRA processName='" + processName + "'");
-        return (processName != null) && processName.equals(ACRA_PRIVATE_PROCESS_NAME);
+        return (processName != null) && processName.endsWith(ACRA_PRIVATE_PROCESS_NAME);

```

Comment:
OK, just add an inline comment to that effect so that we know why when looking at it later.